src/main/java/com/skhu/luxuryshop/dto/ProductDto.java

```diff
@@ -0,0 +1,42 @@
+package com.skhu.luxuryshop.dto;
+
+import com.skhu.luxuryshop.entity.ProductEntity;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class ProductDto {
+    private long id;
+
+    private String productName;
+    private String productPrice;
+    private String productCategory;
```

Comment:
클래스 이름에서 명확하게 어떤 역할을 하는 객체인지 표현해준다면
인스턴스 변수 이름이 조금 간단해질 수 있을 것 같아요!
```suggestion
    private String name;
    private String price;
    private String category;
```